AGGREGATE.md

```diff
@@ -150,12 +142,12 @@ The scheme above will generate the following abstract histogram contributions:
 [
 // campaignCounts
 {
-  key: 1382, // 0b10101100110
+  key: 0x566, // 0b10101100110
```

Comment:
It looks like the final key is incorrect for the given example.
source_key: 0x159
trigger_key: 0x400
final_key: 0x559

The difference from the original design is that currently the offset is from the lowest bit whereas it's from the highest bit before.

Reply:
Oh, great catch! Let me update all the comments here...